exchanges/okx/okx_test.go

```diff
@@ -2986,7 +2864,11 @@ func TestWsPlaceMultipleOrder(t *testing.T) {
 	} else if len(pairs) == 0 {
 		t.Skip("no pairs found")
 	}
-	if _, err := ok.WsPlaceMultipleOrder(resp.Arguments); err != nil {
+	args, okay := resp.Arguments.([]PlaceOrderRequestParam)
+	if !okay {
+		t.Fatal("unexpected argument")
```

Comment:
I am hitting this. You may wish to make this more clear? I'm not sure why the test is failing